src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -68,6 +70,19 @@ internal class ManifestDocument
 				doc.Root.SetAttributeValue (androidNs + "versionCode", value);
 			}
 		}
+		public string MinimumSdk {
+			get {
+				var uses = doc.Root.Element ("uses-sdk");
+				if (uses?.Attribute (androidNs + "minSdkVersion") == null) {
```

Comment:
Given that you actually want the `minSdkVersion` attribute value, why not look for that instead of the element?

    var minAttr = doc.Root.Element ("uses-sdk")?.Attribute (androidNs + "minSdkVersion");
    if (minAttr == null) {
        ...
    }
    return minAttr.Value;